packages/checkout-extension/src/ExtensionProvider.tsx

```diff
@@ -9,15 +9,17 @@ import { ExtensionService } from './ExtensionService';
 
 export interface ExtensionState {
     isShowingLoadingIndicator: boolean;
+    shippingFormRenderTimestamp: undefined | number;
```

Comment:
Why do we need a timestamp?

Reply:
We need a way to trigger Formik's `setValues` method to update Formik's internal context.
In my testing, the component does re-render when the checkout state is updated, however, Formik's context stays the same.
Therefore, using a timestamp to manually trigger two Formik's contexts update(one for address, one for shipping option).

Reply:
Thanks for the explanation, must be due to the way we heavily optimise formik and caching. 
Also doesn't help this lib outdated as well 😢